### ISSUE TITLE
The v0.8.24 output "Could not find or load main class org.jruby.Main" error on Windows

### ISSUE THREAD

**Issue by hiroyuki-sato** [embulk/embulk#675]
* embulk 0.8.24
* OS: Windows (10,2012 Server R2)

### Expect behavior

no error

```
D:\embulk-test>embulk
Embulk v0.8.24
Usage: embulk [-vm-options] <command> [--options]
Commands:
```

### Actual behavior

```
D:\embulk-test>embulk
Error: Could not find or load main class org.jruby.Main
Embulk v0.8.24
Usage: embulk [-vm-options] <command> [--options]
Commands:
```

### Note

* `embulk.bat`: OK(No Error)
* `embulk`: NG

Reference
https://gist.github.com/kazup0n/6d960a808f0fd3eb1848914893553824

**Comment by dmikurube** [comment]
#676 will fix this. :)

**Comment by dmikurube** [comment]
Released v0.8.25, and the issue should be fixed there. Please reopen this issue if the problem is still there.
